{
  "_id": {
    "$oid": "6071e39d43dd71267de1c3f3"
  },
  "author_association": "CONTRIBUTOR",
  "issue_url": "https://api.github.com/repos/pydanny/cookiecutter-django/issues/1118",
  "updated_at": "2017-04-12T16:57:07Z",
  "performed_via_github_app": null,
  "html_url": "https://github.com/pydanny/cookiecutter-django/issues/1118#issuecomment-293642133",
  "created_at": "2017-04-12T16:57:07Z",
  "id": 293642133,
  "body": "@jayfk agreed!\r\n\r\nAny ideas why CI hasn't raised this? Is it because we don't generate the PyCharm files?",
  "user": {
    "gists_url": "https://api.github.com/users/hackebrot/gists{/gist_id}",
    "repos_url": "https://api.github.com/users/hackebrot/repos",
    "following_url": "https://api.github.com/users/hackebrot/following{/other_user}",
    "starred_url": "https://api.github.com/users/hackebrot/starred{/owner}{/repo}",
    "login": "hackebrot",
    "followers_url": "https://api.github.com/users/hackebrot/followers",
    "type": "User",
    "url": "https://api.github.com/users/hackebrot",
    "subscriptions_url": "https://api.github.com/users/hackebrot/subscriptions",
    "received_events_url": "https://api.github.com/users/hackebrot/received_events",
    "avatar_url": "https://avatars.githubusercontent.com/u/6024445?v=4",
    "events_url": "https://api.github.com/users/hackebrot/events{/privacy}",
    "html_url": "https://github.com/hackebrot",
    "site_admin": false,
    "id": 6024445,
    "gravatar_id": "",
    "node_id": "MDQ6VXNlcjYwMjQ0NDU=",
    "organizations_url": "https://api.github.com/users/hackebrot/orgs"
  },
  "url": "https://api.github.com/repos/pydanny/cookiecutter-django/issues/comments/293642133",
  "node_id": "MDEyOklzc3VlQ29tbWVudDI5MzY0MjEzMw=="
}